Utilities/Files.cs

```diff
@@ -23,35 +23,35 @@ public static string Read(string name)
                 }
                 catch (ArgumentException ex)
                 {
-                    Logging.Error("Failed to read from " + name, ex);
+                    Logging.Debug("Failed to read from " + name, ex);
```

Comment:
I think I would like to continue reporting these in production code, they are defiitely the sort of error that we need to know about.

Reply:
I believe this error is getting thrown whenever a user first installs the program. We should mitigate that somehow I think. Thoughts?

Reply:
Good insight. Could I trouble you to open a new issue on that please? I'm a bit knee-deep in localization WIP right now.

Reply:
If you cd pls summarize the file paths for which this occurs upon 1st install, we can evaluate mitigations like having the installer create empty files in the right places.

Alternatively we can think about having the client code pass a "quiet=true" bool to Files.Read() that tells it not to log.